quiz.js

```diff
@@ -36,7 +42,12 @@ db.dropDatabase();
 // Jon Arryn
 
 // your code begins here
-
+db.houses.update( {name: 'Stark'},
+  { $push: {people: 'Ned Stark, Arya Stark, Sansa Stark'}});
+db.houses.update( {name: 'Targaryen'},
+  { $push: {people: 'Viserys Targaryen, Daenerys Targaryen'}});
+db.houses.update( {name: 'Arryn'},
+  { $push: {people: 'Jon Arryn'}});
 
 
```

Comment:
Not quite. `'Ned Stark, Arya Stark, Sansa Stark'` is a single string. While $push won't complain about this, since it's expecting a single value, it means that if we added another person, say 'Jon Snow', to House Stark, we'd end up with the following.
```
{
	"_id" : ObjectId("..................................."),
	"name" : "Stark",
	"motto" : "Winter is Coming",
	"people" : [
		"Ned Stark, Arya Stark, Sansa Stark",
		"Jon Snow"
	]
}
```
The correct approach would have been either to $push each of these people individually or to use $each to add them all together, like this:
```
db.houses.update({name: 'Stark'}, { $push: { people: { $each: [
  'Ned Stark', 
  'Arya Stark', 
  'Sansa Stark'
]}}});
```